encoding.bs

```diff
@@ -1923,8 +1923,7 @@ consumers of content generated with <a>GBK</a>'s <a for=/>encoder</a>.
 
    <li><p>Unset the <a>EUC-JP jis0212 flag</a>.
 
-   <li><p>If <var>byte</var> is not in the range 0xA1 to 0xFE, inclusive,
-   <a>prepend</a> <var>byte</var> to
+   <li><p>If <var>byte</var> is an <a>ASCII byte</a>, <a>prepend</a> <var>byte</var> to
```

Comment:
I don't see why we have to check that code point is null. We already know it's null because byte is an ASCII byte. (That doesn't work with EUC-KR because ASCII byte is inside the valid range.)

I would be okay with changing order though, I guess I should look at how much that would affect.